froide/document/templates/filingcabinet/_set_description_form.html

```diff
@@ -0,0 +1,8 @@
+{% extends './_base_set_form.html' %}
```

Comment:
We have not used relative template paths anywhere yet as far as I'm aware, but I can see their use case.